src/features/user/state/ui/CouchDB.saga.ts

```diff
@@ -0,0 +1,27 @@
+import { takeLatest, select, call } from 'redux-saga/effects';
+import { getType } from 'typesafe-actions';
+import { syncSaga } from 'sagas/sync';
+import { loadSetting } from 'sagas/settings';
+import { getCouchDBSetting } from './CouchDB.selector';
+import * as CouchDB from './CouchDB.action';
+import { isUserLoggedIn } from '../User.saga';
+
+export function* finishCouchDBSettingSaga() {
+  const couchDB = yield select(getCouchDBSetting);
+  const userInfo = {
+    'accessToken': '',
+    "couchDB": { ...couchDB,
+      "transactions": couchDB.url + "/money_tracker_transactions_" + couchDB.username,
+      "accounts": couchDB.url + "/money_tracker_accounts_" + couchDB.username,
+      "settings": couchDB.url + "/money_tracker_settings_" + couchDB.username,
```

Comment:
there's also `tags` collection we need, along with accounts/settings/transactions

Reply:
I have one transaction with tags in my test data, but didn't didn't see this collection after I changed my testing data to sync with private CouchDB server.

Maybe something went wrong?

Reply:
hm, it should be created in initial setup. Could you try to sign in again from incognito tab to see if it's created after sync?

Reply:
It's my fault. After the tags database being added to localstorage, the tags database is now synced.